bankaccount/client.go

```diff
@@ -108,15 +107,7 @@ func Del(id string, params *stripe.BankAccountParams) (*stripe.BankAccount, erro
 
 func (c Client) Del(id string, params *stripe.BankAccountParams) (*stripe.BankAccount, error) {
 	ba := &stripe.BankAccount{}
-	var err error
-
-	if len(params.Customer) > 0 {
-		err = c.B.Call("DELETE", fmt.Sprintf("/customers/%v/bank_accounts/%v", params.Customer, id), c.Key, nil, nil, ba)
-	} else if len(params.AccountID) > 0 {
-		err = c.B.Call("DELETE", fmt.Sprintf("/accounts/%v/bank_accounts/%v", params.AccountID, id), c.Key, nil, nil, ba)
-	} else {
-		err = errors.New("Invalid bank account params: either customer or AccountID need to be set")
-	}
+	err := c.B.Call("DELETE", fmt.Sprintf("/accounts/%v/bank_accounts/%v", params.AccountID, id), c.Key, nil, nil, ba)
```

Comment:
It's the only function in the client that handled Customers, I made an educated assumption. It should have been in another commit though